system-tests/lib/cre/crib/crib.go

```diff
@@ -271,12 +332,12 @@ func nodesetDockerImage(nodeSet *types.CapabilitiesAwareNodeSet) (string, error)
 		}
 
 		// TODO use kubectl cp to copy them?
-		if len(nodeSpec.Node.CapabilitiesBinaryPaths) > 0 {
-			return "", fmt.Errorf("capabilities binaries are not supported in CRIB. Please use a Docker image that already contains the capabilities and remove capabilities_binary_paths from the node spec at index %d in nodeSet %s", nodeIdx, nodeSet.Name)
-		}
-		if nodeSpec.Node.CapabilityContainerDir != "" {
-			return "", fmt.Errorf("capabilities binaries are not supported in CRIB. Please use a Docker image that already contains the capabilities and remove capability_container_dir from the node spec at index %d in nodeSet %s", nodeIdx, nodeSet.Name)
-		}
+		// if len(nodeSpec.Node.CapabilitiesBinaryPaths) > 0 {
```

Comment:
Let's remove it then if we decide not to go with kubectl cp and use devspace command